docs/source/routing/errors.mdx

```diff
@@ -71,9 +71,41 @@ You can create Rhai scripts that throw custom status codes. See [Terminating cli
 
 </Note>
 
-## GraphQL error codes
+## GraphQL spec error codes
 
-GraphQL error codes can appear in client responses under `errors[].extensions.code`, which is an established convention found in [GraphQL error extensions](https://spec.graphql.org/October2021/#sec-Errors.Error-result-format). Learn how to see these error codes in Studio via [extended error metrics](/graphos/routing/graphos-reporting#enabling-extended-error-reporting).
+These error codes are defined in the [GraphQL spec](https://spec.graphql.org/October2021/) and can appear in client responses under `errors[].extensions.code`, which is an established convention found in [GraphQL error extensions](https://spec.graphql.org/October2021/#sec-Errors.Error-result-format). Because these error codes are part of the core spec, they cannot be disabled directly as they are part of the core Router implementation. Only Apollo Router-specific features or subgraph errors can be disabled.
+
+Changing these errors makes your router non-compliant with the GraphQL spec. To edit the response, you must use an external proxy.
```

Comment:
@smyrick the last sentence is unclear to me. What do we mean by "edit the response"? Can we link out to anything there for more information?